content/en/users/tutorials/htc-job-submission/_index.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Submitting HTC Jobs"
+type: docs
+weight: 160
+description: >
+  Submitting High Throughput Compute jobs
+---
+
+## Overview
+
+This tutorial describes how to submit
+[High Throughput Compute (HTC)](../../compute/high-throughput-compute) jobs
+using command-line.
+
+## Prerequisites
+
+To submit an EGI HTC job, you will have to:
+
+1. Obtain an X.509 certificate. The certificates are issued by Certification
+   Authorities (CAs) part of the
+   [European Policy Management Authority for Grid Authentication](https://www.eugridpma.org)
+   (EUGridPMA), which is also part of the
+   [International Global Trust Federation](https://www.igtf.net) (IGTF).
+1. Enrol into a VO having access to HTC resources.
+
+> This tutorial will be using `dteam` a test Virtual Organisation that can be
+> used by resource providers, commands should be adjusted to the appropriate VO.
+
+## Step 1: getting access to a User Interface (UI)
+
+In order to interact with HTC resources, you should have access to a
+`User Interface`, often referred to as a `UI`. This software environment will
+provide all the tools required to interact with the different middleware, as
+different sites can be using different Computing Element (CE), such as
+HTCondorCE and ARC-CE (CREAM is a legacy software stack that is now more
+officially supported).
+
+Different possibilities are available to access an UI:
+
+- Having access to an UI provided by/for your community
+- Deploying a UI
+
+### Deploying an UI
+
+The UI is available as a package in the [UMD](https://go.egi.eu/umd) software
+distribution, but it will also required additional software and configuration.
+
+In order to help with deploying an UI, different solutions are possible:
+
+- Some
+  [Ansible roles](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html)
+  are available in the
+  [EGI Federation GitHub organisation](https://github.com/EGI-Federation?q=ansible-role),
+  mainly [ansible-role-ui](https://github.com/EGI-Federation/ansible-role-ui)
+  that should be used together with
+  [ansible-role-VOMS-client](https://github.com/EGI-Federation/ansible-role-VOMS-client),
+  providing software and material required for the authentication and
+  authorisation, and
+  [ansible-role-umd](https://github.com/EGI-Federation/ansible-role-umd)
+  configuring the software repositories from where all the software will be
+  installed.
+- The repository
+  [ui-deployment](https://github.com/EGI-Federation/ui-deployment) provides an
+  [terraform](https://terraform.io) based deployment allowing to deploy a
+  `User Interface (UI)` in a
+  [Cloud Compute virtual machine](../../compute/cloud-compute). This integrated
```

Comment:
I don't understand what do you mean with this last phrase

Reply:
Yes, apparently missing some words :)
Will recheck/update this part, and add a note on setting it up manually, thanks!

Reply:
I've updated this part, is it better now?

Reply:
good!